ddtrace/internal/_telemetry.py

```diff
@@ -0,0 +1,36 @@
+import ddtrace.internal.telemetry
```

Comment:
Is there an actual need for this API as opposed to the one in `ddtrace.internal.telemetry`? I think we could just stick with the `enable`/`disable` methods and avoid the additional indirection of this class.

Reply:
I removed this file. I was following the RuntimeMetrics API